login-fire-form.html

```diff
@@ -287,9 +292,9 @@
     _signUp: function(e) {
         this._cleanMessages();
         this.$$("#repeatPasswordInput").invalid=false;
-        e = this.$.emailInput.validate()
-        p = this.$.passwordInput.validate()
-        pRepeat = this.password == this.passwordAgain
+        e = this.$$("#emailInput").validate()
```

Comment:
I can't understand why selection by ID is breaking something. It supposed to be scoped. The shortcut for _querySelection_ will not be removed from Polymer v2. In fact I don't know. It was a question.